### DISCUSSION TITLE
How to add conflicted enchants/ and how to add more then the the max

### DISCUSSION THREAD

**Discussion by Deadsereis** [Geolykt/EnchantmentsPlus#125] [Q&A]
Hi, i'm a server owner and i want to make is so conflicted enchants work in 1.17.1 if that doesn't work its fine, i really just need to know how to add more then the max amount of enchantments. Thank you.

**Comment by Geolykt** [comment]
If you want ordinary players to add the enchantments, simply change the relevant world configuration files. For the command use `/ench <enchantment> <level> <modifier> <doNotification> <force>`, see https://github.com/Geolykt/EnchantmentsPlus#commands
